include/SDL3/SDL_begin_code.h

```diff
@@ -227,3 +225,8 @@
 #define SDL_ALLOC_SIZE2(p1, p2)
 #endif
 #endif /* SDL_ALLOC_SIZE2 not defined */
+
+/* This is used to mark functions that return memory that need to be freed with SDL_free() */
+#ifndef SDL_DECLSPEC_FREE
+#define SDL_DECLSPEC_FREE   SDL_DECLSPEC SDL_MALLOC
```

Comment:
This isn't what SDL_MALLOC does. It doesn't mean "this allocates some memory that needs to be freed," but rather "this is an _allocator_ function like malloc with certain rules." Among them, that the allocated memory is _uninitialized_ and the compiler can optimize for that truth.

At least according to https://stackoverflow.com/questions/18485447/gcc-attribute-malloc

Reply:
Oh, okay, fixing shortly.